vspace/src/main/java/edu/asu/diging/vspace/references/ReferenceDisplayFormatter.java

```diff
@@ -0,0 +1,67 @@
+package edu.asu.diging.vspace.references;
```

Comment:
there shouldn't be any other packages than `config`, `core`, and `web` in `edu.asu.diging.vspace`. All classes that are not config or controller classes need to be in a subpackage of `core`.